node/src/main/scala/com/wavesplatform/block/serialization/BlockSerializer.scala

```diff
@@ -0,0 +1,118 @@
+package com.wavesplatform.block.serialization
+
+import java.nio.ByteBuffer
+
+import com.google.common.primitives.{Bytes, Ints, Longs, Shorts}
+import com.wavesplatform.block.Block.{NgBlockVersion, ProtoBlockVersion, RewardBlockVersion}
+import com.wavesplatform.block.{Block, BlockHeader}
+import com.wavesplatform.common.state.ByteStr
+import com.wavesplatform.crypto.SignatureLength
+import com.wavesplatform.protobuf.block.PBBlocks
+import com.wavesplatform.serialization.ByteBufferOps
+import com.wavesplatform.transaction.Asset.Waves
+import play.api.libs.json.{JsArray, JsNumber, JsObject, Json}
+
+import scala.util.Try
+
+object BlockHeaderSerializer {
+  def toJson(blockHeader: BlockHeader): JsObject = {
+    val consensusJson =
```

Comment:
`consensusJson` as defined here should only be used for legacy block versions. For versions 5+, both fields should be named in camelCase and defined at the top level